scripts/nolints.txt

```diff
@@ -33,7 +33,7 @@ apply_nolint euclidean_domain.xgcd_aux doc_blame
 apply_nolint division_ring doc_blame
 apply_nolint field doc_blame
 
--- algebra/gcd_domain.lean
+-- algebra/gcd_monoid.lean
```

Comment:
Alternatively, just write the three docstrings? :grinning:

Reply:
Sorry, I have no idea how nolints works, my find-and-replace just caught it... which docstrings are needed?

Reply:
Docstrings on the three definitions listed below it.

Reply:
FYI, this file keeps track of "old" definitions that don't have docstrings because they were written before we started caring about docstrings. One of our goals is to make this file become empty. You can track the progress here: https://leanprover-community.github.io/mathlib_stats/nolints.png

Reply:
Yeah, this file should almost never be edited manually. Our CI will take care of updates to this.

Reply:
Please revert the changes to `nolints.txt`.

Reply:
I've tried - it's marking one line as changed even though it seems to be the same before and after.

Reply:
Oh, that's probably the lack of a newline at the end of the file. That's good enough.